py_env_create

```diff
@@ -20,6 +20,7 @@ if [ $error != 0 ]; then
    echo "For notes on installing on a user system see: https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html"
```

Comment:
These changes get moved over the the github action branch, and removed from here.